cegs_portal/search/views/v1/search.py

```diff
@@ -455,6 +464,17 @@ def request_options(self, request):
         return options
 
     def get(self, request, options, data):
+        if request.headers.get("HX-Target"):
```

Comment:
Instead of checking only for the existence of HX-Target you should also check what the value is. That feels more "secure" or "sturdy" to me.

Reply:
```suggestion
        if (target := request.headers.get("HX-Target")) is not None and target == "feature_sigreo-table":
```